Sources/ApolloCodegenLib/Templates/EnumTemplate.swift

```diff
@@ -25,9 +25,22 @@ struct EnumTemplate: TemplateRenderer {
   }
 
   private func evaluateDeprecation(graphqlEnumValue: GraphQLEnumValue) -> String? {
```

Comment:
Can we change the name of this function to `EnumCaseTemplate(_ graphQLEnumValue: GraphQLEnumValue)`.

This is returning the template string to render, not a value indicating an evaluation of deprecation reason.